app/Models/Facturer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Facturer extends Model
+{
+    use HasFactory;
+    protected $table = 'manufacturer';
+    protected $filltable = ['manufacturer_name'];
```

Comment:
Each element in the array must have a newline and a comma at the end of each element